unfetter-discover/unfetter-ui
# Updates to "Where are you most vulnerable chart"

Delinkify Assessed Item for Capability assessments; increase size of Risk value. Remove temporary coming soon divs.

To Test:
- Go to the Assessments beta summary page for a capability assessment.
- In the "Where are you most vulnerable" chart, the ASSESSED ITEM should not be a clickable link.
- The RISK value should be an easily readable size.
- Switch to any non-capability assessment.
- In the "Where are you most vulnerable" chart, the ASSESSED ITEM **should be** a clickable link.
- The RISK value should be an easily readable size.
